lib/native-client.js

```diff
@@ -106,6 +106,56 @@ class NativeClient {
   }
 
   /**
+   * Get the currentOp.
+   *
+   * @param {Integer} includeAll - if true also list currently idle operations in the result.
+   * @param {Function} callback - The callback.
+   */
+  currentOp(includeAll, callback) { // TODO: deal with db versions? && do we want operations arg?
+
+    this.database.admin().command({'currentOp': 1, '$all': includeAll}, (error, result) => {
```

Comment:
I tried breaking up the lines but got a bunch of complains from the githook, is there a style guide?